.github/scripts/nova_postscript.bash

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+echo "Current working directory: $(pwd)"
+cd "$FBGEMM_REPO" || echo "Failed to cd to $FBGEMM_REPO"
+PRELUDE=.github/scripts/setup_env.bash
+BUILD_ENV_NAME=base
+
+# Install FBGEMM_GPU Nightly
+echo "Current working directory: $(pwd)"
+# shellcheck disable=SC1091
+# shellcheck source=.github/scripts/setup_env.bash
+. $PRELUDE; install_fbgemm_gpu_package "$BUILD_ENV_NAME" fbgemm_gpu/dist/*.whl
```

Comment:
Looks like the job is now failing in the post-script phase. Is this script looking for the wheel in the incorrect directory?

Reply:
The directory is correct, since I already cd "$FBGEMM_REPO". The wheel should be in fbgemm_gpu/dist. I think the problem is the building wheel step, so I'm fixing that.